osbs/core.py

```diff
@@ -872,6 +872,23 @@ def restore_resource(self, resource_type, resource):
         check_response(response)
         return response
 
+    def create_config_map(self, config_data):
+        url = self._build_k8s_url("configmaps/")
+        response = self._post(url, data=json.dumps(config_data))
+        check_response(response)
+        return response
+
+    def get_config_map(self, config_name):
+        url = self._build_k8s_url("configmaps/")
```

Comment:
Need to include the name in the url, otherwise we'll get a v1.ConfigMapList of all config maps: https://docs.openshift.org/latest/rest_api/kubernetes_v1.html#list-or-watch-objects-of-kind-configmap